### PR TITLE
Update and fix of dotnet core project

### PR DESCRIPTION
**Pull request by ondraslim**
Updated dotnet core and dotvvm version, added example setup for 'code' response type, updated readme

### CODE REVIEW

File: AspNetCore/DotvvmAuthSample/DotvvmAuthSample/Startup.cs [hunk 1]
```diff
@@ -47,10 +47,18 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClientId = Configuration["AzureAD:ClientId"];
                 options.Authority = string.Format(Configuration["AzureAd:AadInstance"], Configuration["AzureAd:Tenant"]);
                 options.SignedOutRedirectUri = Configuration["AzureAd:PostLogoutRedirectUri"];
+                //options.ResponseType = "code";
```
Comment (tomasherceg) [inline AspNetCore/DotvvmAuthSample/DotvvmAuthSample/Startup.cs]: I'd change this to something like that:

```
// Uncomment the following section if you need "code" response type
// options.ResponseType = "code";
// options.ClientSecret = "Configuration["AzureAd:ClientSecret"];
```

File: AspNetCore/DotvvmAuthSample/DotvvmAuthSample/DotvvmAuthSample.csproj [hunk 2]
```diff
@@ -24,10 +24,10 @@
         <Folder Include="wwwroot\" />
     </ItemGroup>
     <ItemGroup>
-        <PackageReference Include="Microsoft.AspNetCore" Version="2.0.3" />
-        <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.3" />
-        <PackageReference Include="DotVVM.AspNetCore" Version="2.0.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="2.0.0" />
+        <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
```
Comment (tomasherceg) [inline AspNetCore/DotvvmAuthSample/DotvvmAuthSample/DotvvmAuthSample.csproj]: I think that from .NET Core 3.0, the references to `Microsoft.AspNetCore.*` are not needed anymore - they should be already referenced by the `Microsoft.NET.Sdk.Web`. 
Can you please remove all three packages (keep only `DotVVM.AspNetCore` and try whether it works?

File: AspNetCore/DotvvmAuthSample/DotvvmAuthSample/Startup.cs (hunk 1 above)
Comment (ondraslim) [reply AspNetCore/DotvvmAuthSample/DotvvmAuthSample/Startup.cs]: Good point, added this to the source code

File: AspNetCore/DotvvmAuthSample/DotvvmAuthSample/DotvvmAuthSample.csproj (hunk 2 above)
Comment (ondraslim) [reply AspNetCore/DotvvmAuthSample/DotvvmAuthSample/DotvvmAuthSample.csproj]: Microsoft.AspNetCore.Authentication.OpenIdConnect seems to be required. The sample app works without them.

**Pull request merged**